package.json

```diff
@@ -43,61 +43,60 @@
   },
   "devDependencies": {
     "#": "link:./src/filters",
-    "@11ty/eleventy": "^0.10.0",
+    "@11ty/eleventy": "^0.11.0",
```

Comment:
@mirisuzanne Check out the release notes: https://github.com/11ty/eleventy/releases/tag/v0.11.0

The one I notice is that `renderData` is deprecated, replaced by https://www.11ty.dev/docs/data-computed/